src/viur/core/modules/user.py

```diff
@@ -218,7 +218,7 @@ def __init__(self, moduleName, modulePath, userModule):
         super().__init__(moduleName, modulePath)
         self._user_module = userModule
 
-    def can_handle(self, user: db.Entity) -> bool:
+    def can_handle(self, skel: skeleton.SkeletonInstance) -> bool:
```

Comment:
The other `can_handle` functions have `db.Entity` as type hint. Shouldn't that also be changed